### ISSUE TITLE
Opening and closing audio cues picker adds a lot of settings

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#207946]
I opened the picker to see all audio cues and just clicked OK, my settings then got all of this:

![image](https://github.com/microsoft/vscode/assets/900690/7a568503-eb95-4118-add3-8d10dbff89b5)

```
"accessibility.signals.lineHasError": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.lineHasWarning": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.lineHasFoldedArea": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.lineHasBreakpoint": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.lineHasInlineSuggestion": {
    "sound": "off"
  },
  "accessibility.signals.terminalQuickFix": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.onDebugBreak": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.noInlayHints": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.taskCompleted": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.taskFailed": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.terminalCommandFailed": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.terminalBell": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.notebookCellCompleted": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.notebookCellFailed": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.diffLineInserted": {
    "sound": "off"
  },
  "accessibility.signals.diffLineDeleted": {
    "sound": "off"
  },
  "accessibility.signals.diffLineModified": {
    "sound": "off"
  },
  "accessibility.signals.chatRequestSent": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.chatResponseReceived": {
    "sound": "off"
  },
  "accessibility.signals.chatResponsePending": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.clear": {
    "sound": "off",
    "announcement": "auto"
  },
  "accessibility.signals.voiceRecordingStopped": {
    "sound": "off"
  }
```

I would have not expected any settings change unless I change away from the default?

**Comment by meganrogge** [comment]
Hmm, maybe I need to pass a configuration target for this to happen?

https://github.com/microsoft/vscode/blob/b0d975fc6370f15570c380f41015b1ca2cdcb791/src/vs/platform/configuration/common/configuration.ts#L154
